index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.2/css/bootstrap.css">
+    <title>Student Grade Table</title>
+  </head>
+  <body>
+    <h2>Student Grade Table</h2>
+    <table class="table table-striped table-bordered">
```

Comment:
The `table` class is unnecessary here. Bootstrap only uses `.table` as a way to style non-`<table>` elements like a table.